### PR TITLE
Add githubstudent's pin to the map

### LINKED ISSUE #5048: Please add me
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by IsseOmar**
add townsville to the 

 closes #5048

### CODE REVIEW

**Comment by califocation** [comment]
your file name should be USERNAME.json

File: _pins/githubstudent.townsville [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubhandle:githubstudent
+longitudea:146.816948
+latitude:-19.258964
+---
```
Comment (hectorsector) [inline _pins/githubstudent.townsville]: @IsseOmar 
Great work so far! You'll need to make sure that your code matches exactly the template in the class, including `---` before and after the code:
```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```

Additionally, your file name should be `_pins/isseomar.json`

**Comment by hollenberry** [comment]
Hey @IsseOmar  ! Looks like you've gotten good advice from Hector above! If you take action on those changes, you'll be set. 👍

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.

**Comment by crichID** [comment]
@IsseOmar I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/5050/commits/248905bedd3ebbd316c215a760abf38cdaddd175. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 21st, I will go ahead and do it just to keep the repository clean!

**Pull request merged**